src/containers/shipment/shipment-creation/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { ShipmentService } from '../../../services/shipment-service';
+import { useForm } from 'react-hook-form';
+import { FormItemWrapper, FormWrapper } from './index.styles';
+
+interface ShipmentCreationContainerProps {
+  shipmentService: ShipmentService;
+}
+
+export const ShipmentCreationContainer: React.FC<ShipmentCreationContainerProps> = ({ shipmentService }) => {
+  const { handleSubmit } = useForm<FormData>();
+
+  console.log({ shipmentService });
+  const onSubmit = () => {};
+  return (
+    <FormWrapper>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <FormItemWrapper>
+          <label>Starting Location:</label>
+          <input name="starting_location" />
+          {/* {errors.starting_location && <span>This field is required</span>} */}
+        </FormItemWrapper>
+
+        <FormItemWrapper>
+          <label>Destination Location:</label>
+          {/* TODO this needs to be a dropdown */}
```

Comment:
Nitpick: can we create issues and reference them in the TODO comments? It then means we know all the things we need to do and it produces a nice list of small bits to pick up when we have the time.